app-backend/ingest/build.sbt

```diff
@@ -0,0 +1,17 @@
+name := "rf-ingest"
+
+assemblyJarName in assembly := "rf-ingest.jar"
+
+mainClass in (Compile, assembly) := Some("com.azavea.rf.ingest.Ingest")
+
+javaOptions += "-Xmx8G"
```

Comment:
Does this still need to take up 8GB if we're working with smaller tiles right now in development